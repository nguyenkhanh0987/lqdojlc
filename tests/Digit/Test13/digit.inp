100
608807760 613972801
705883063 836031565
861308922 954644001
502649189 900260602
611219510 788239229
373671755 763506757
450948793 631919150
716849338 959732068
464657151 588083925
257446905 610471645
423078584 799535416
857143850 914410869
740648426 911340929
682838714 856349106
406767431 835784674
706104831 958470667
900304632 995543764
904260176 949324622
584534203 762206686
311069657 405514593
90757793 170367716
920934329 926509483
829112297 995904546
752499611 821732639
151521883 179719654
593170791 747160147
526049033 594929602
324637757 536105583
335244759 795087900
298381423 733477588
139000690 600041189
284170356 550178602
958150523 965675999
195759874 471680712
9474454 840957406
945800647 949193612
662324451 719040066
605204739 729259769
58530739 197509728
279379725 611005233
840045764 961377127
12646479 76531192
214002794 652385687
418971114 983178681
966519178 980034812
193469269 771480567
618338716 657807625
928017392 938294627
51566145 955720421
259973019 293536965
669296251 876124961
772266443 819457253
103800995 345808339
405386191 412560531
229167097 977367108
581651497 896833519
747490005 999375234
481626075 776284428
597272318 653074979
551178918 979581784
819398220 926656945
65049127 699829482
387759406 734994356
263871122 746324723
884986948 938230047
327125166 443722189
495280985 752926744
628827426 689505642
38484303 84470489
263590096 816765963
475779104 672124768
285367515 357665837
210802296 959084068
197422492 564999825
645854689 943484780
505109270 539741351
515891023 670418296
629857446 852970672
856245435 987780316
140391695 201515827
529753153 888272755
316544147 411447844
378960838 496326655
44205475 492741933
321855872 612284320
831100114 945736978
117725621 444145122
958087613 969811847
806369594 991055775
225389873 791345598
887958056 946525507
378854128 477214550
220479949 338672078
292365363 830572580
559783549 800112928
583098493 991123150
51277937 968756195
204386945 486868330
839891188 882020559
428332141 437954207
