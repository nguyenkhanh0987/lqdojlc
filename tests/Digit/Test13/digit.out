336991
9699263
6321668
37767792
16634940
45295662
22477069
17393619
16356665
50906157
42529170
4989378
14345981
14004469
45102704
17741768
5654959
1376944
16714673
15007852
14173228
191451
12098452
6483554
6203136
13759464
9420697
29283188
55731036
52929667
72196990
36613177
602634
41716014
117653864
73858
6316597
11029696
29559419
46535804
8182371
12883605
60174421
54547510
1000826
74777203
3172290
367271
118029368
7778051
17977242
4342798
40865649
447485
84874053
29311932
19495770
31245762
3932575
37723522
7393540
96856249
40035746
61730098
2819383
16844602
25777950
6903646
10858106
68619298
22453571
10223608
85450507
52886171
24578321
2290652
16305107
19759972
9366398
14006410
34684319
14760880
17782734
78007488
39569018
7697558
53815936
752283
12049052
73005787
2978694
13834377
19122996
62572828
25797491
34196781
118925917
43167511
3669211
1023782
